cdap-ui/app/services/helpers.js

```diff
@@ -96,11 +96,29 @@ angular.module(PKG.name+'.services')
     return obj;
   }
 
+   function formatDate(date) {
+      var hours = date.getHours();
+      var minutes = date.getMinutes();
+      var seconds = date.getSeconds();
+      var ampm = hours >= 12 ? 'PM' : 'AM';
+      hours = hours % 12;
+      hours = hours ? hours : 12;
+      minutes = minutes < 10 ? '0' + minutes : minutes;
+      seconds = seconds < 10 ? '0' + seconds : seconds;
+      // example format: 1:09:11 PM
+      return hours + ':' + minutes + ':' + seconds + ' ' + ampm;
+    }
+
+    function dateToString(e) {
+      var date = new Date(e * 1000);
+      return formatDate(date);
+    }
```

Comment:
Can you call the Angular Moment filter like a function? and use the value in other javascript?